internal/rbd/rbd_util.go

```diff
@@ -451,7 +451,7 @@ func (rv *rbdVolume) isInUse() (bool, error) {
 // addRbdManagerTask adds a ceph manager task to execute command
 // asynchronously. If command is not found returns a bool set to false
 // example arg ["trash", "remove","pool/image"].
-func addRbdManagerTask(ctx context.Context, pOpts *rbdVolume, arg []string) (bool, error) {
+func addRbdManagerTask(ctx context.Context, pOpts *rbdVolume, arg []string) (bool, string, error) {
```

Comment:
Instead of returning an additional string, can the `stderr` text not be included in the returned error? That would keep things simpler, I think.

Reply:
I can do that.